scripts/Calibration/tube.py

```diff
@@ -424,12 +454,13 @@ def calibrate(ws, tubeSet, knownPositions, funcForm, **kwargs):
         #
         # Get tube length and number of detectors
         tube_length = tubeSet.getTubeLength(0)
-        #ndets = len(wsp_index_for_tube0)
+        # ndets = len(wsp_index_for_tube0)
+        # pylint:disable= unused-variable
```

Comment:
Rather than disabling `unused-variable` the ones not used should be removed. If it's one of the `id1, ndets, step` just down here you can replace them with the anonymous/throw-away `_`.